mivs/site_sections/mivs_admin.py

```diff
@@ -124,6 +124,15 @@ def edit_judge(self, session, message='', **params):
             'message': message
         }
 
+    def judges_owed_refunds(self, session):
+        return {
+            'judges': [a for a in session.query(Attendee).join(Attendee.admin_account)
+                .filter(AdminAccount.judge != None)
```

Comment:
Note that the relationships here are set up a bit strangely, so that this query ends up returning anyone with a "judge" entry in the MIVS admin section, _regardless of their admin account privileges_. This means that even if you go into an attendee and remove their "Indie Judge" access, they'll still be listed on this page.

Based on the wording I've seen elsewhere, it doesn't look like judges get their free badges revoked for any reason, so it should be fine.

Reply:
This looks cool. Only change I would recommend is incorporating the `if` statements into the query itself:

```python
session.query(Attendee) \
    .outerjoin(Group, Attendee.group_id == Group.id) \
    .filter(or_(
        Attendee.paid == c.HAS_PAID, 
        and_(
            Attendee.paid == c.PAID_BY_GROUP,
            Group.amount_paid > 0))) \
    .join(Attendee.admin_account) \
    .filter(AdminAccount.judge != None) \
    .options(joinedload(Attendee.group)) \
    .order_by(Attendee.full_name)
```

You could also add a check for "Indie Judge" access, but I don't know if that's what we want?
```python
.filter(
    AdminAccount.judge != None, 
    AdminAccount.access.contains(str(c.INDIE_JUDGE)))
```

Reply:
Great suggestion in both cases. :D Done!